sites/www.seafloorexplorer.org.conf

```
@@ -8,5 +6,24 @@ server {
         return 301 https://s3.amazonaws.com/$host$request_uri;
     }
 
-    include /etc/nginx/proxy.conf;
+    location / {
+        resolver 8.8.8.8;
+        proxy_set_header       Host zooniversestatic.z13.web.core.windows.net;
+        proxy_http_version 1.1;
+
+        add_header 'X-Content-Type-Options' 'nosniff';
+        add_header 'Content-Security-Policy' "frame-ancestors 'self' $csp_whitelist";
```

this `$csp_whitelist` is not set in this context so it's blank and won't be used.
```suggestion
        add_header 'Content-Security-Policy' "frame-ancestors 'self'";
```